tools/cobra/main.go

```diff
@@ -63,7 +74,6 @@ func run(args []string) error {
 	for _, c := range []*cobra.Command{
 		cainjectorapp.NewCommandStartInjectorController(nil, nil, nil),
 		controllerapp.NewCommandStartCertManagerController(nil),
-		ctlcmd.NewCertManagerCtlCommand(nil, nil, nil, nil),
 		webhookcmd.NewServerCommand(nil),
```

Comment:
It would be good for us to include the ctl command also IMO. I believe it isn't actually working atm, but I'm not sure why...

Reply:
It does work, it generates a file for each subcommand of the plugin though.
I removed it because there are already more human readable plugin docs [here](https://cert-manager.io/docs/usage/kubectl-plugin/). But perhaps some duplication is ok

Reply:
I think it would be good to have. It means that we have a doc page which is _always_ up to date and has all the information on it. I can see that that page is out of date.